src/Http/StreamedGeneratorResponse.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Baldinof\RoadRunnerBundle\Http;
+
+use Symfony\Component\HttpFoundation\StreamedResponse;
+
+final class StreamedGeneratorResponse extends StreamedResponse
+{
+    public function __construct(iterable|callable $callbackOrChunks, int $status = 200, array $headers = [])
+    {
+        if (is_iterable($callbackOrChunks)) {
+            $callbackOrChunks = fn () => yield from $callbackOrChunks;
+        }
+        parent::__construct($callbackOrChunks, $status, $headers);
+    }
```

Comment:
```suggestion
    /**
     * @param iterable<string>|(callable(): iterable<string>) $callbackOrChunks
     */
    public function __construct(private iterable|callable $callbackOrChunks, int $status = 200, array $headers = [])
    {
        parent::__construct(function () {
            foreach ($this->getGenerator() as $chunk) {
                echo $chunk;
            }
        }, $status, $headers);
    }

    /**
     * @return \Generator<string>
     */
    public function getGenerator(): \Generator {
        if (is_callable($this->callbackOrChunks)) {
            $chunks = ($this->callbackOrChunks)();

            yield from $chunks;
        }

        yield from $chunks;
    }
```

This way when not using RoadRunner Symfony will still stream using echo, and in the worker we can use `getGenerator()` to use the RR stream feature